resource-timing/clear_resource_timing_functionality.html

```diff
@@ -14,9 +14,9 @@
     const context = new PerformanceContext(performance);
     function onload_test()
     {
-        test_equals(context.getEntriesByType('resource').length, 4, 4 + ' resource timing entries should be stored in this page.');
+        assert_equals(context.getEntriesByType('resource').length, 4, 4 + ' resource timing entries should be stored in this page.');
```

Comment:
I'd expect this test no longer has any subtests. Doesn't this result in a single test with a less useful name? Can you get review from someone who was involved with adding all of this test_* stuff?